src/api/spec/controllers/webui/user_controller_spec.rb

```diff
@@ -0,0 +1,184 @@
+require 'rails_helper'
+
+RSpec.describe Webui::UserController do
+  let!(:user) { create(:confirmed_user, login: "tom") }
+  let!(:non_admin_user) { create(:confirmed_user, login: "moi") }
+  let!(:admin_user) { create(:admin_user, login: "king") }
+  let(:deleted_user) { create(:deleted_user) }
+
+  describe "GET #index" do
```

Comment:
@hennevogel @ChrisBr: So doing this https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/controllers/webui/configuration_controller_spec.rb#L8 is a non sense, the checking of the before_action should be outside any "describe" and then inside each action you should check the templated rendered at least